test/event_stream/subscriber_dsl_test.rb

```diff
@@ -0,0 +1,54 @@
+require_relative '../test_helper'
+
+module EventStream
+  class SubscriberDSLTest < Minitest::Should::TestCase
+
+    class MySubscriber
+      include SubscriberDSL
+
+      class << self
+        attr_accessor :events
+      end
+
+      on(:event) { |event| self.events << event  }
+
+      def a_method(event)
+        self.class.events << event
+      end
+    end
+
+    context 'The Subscriber DSL' do
+      setup do
+        MySubscriber.events = []
+      end
+
+      context 'the default stream' do
+        setup do
+          EventStream.clear_subscribers
```

Comment:
This also brings up another questions: can we subscribe to multiple event streams?